test/main.pony

```diff
@@ -88,20 +88,45 @@ class _TestSelect is UnitTest
     let install_args: {(String): Array[String] val} val =
       {(v) => ["update"; "ponyc"; v; "--platform=" + platform] }
 
-    let link =
-      FilePath(
-        h.env.root,
-        "./.pony_test/select/ponyup/bin/ponyc")
+    let link = FilePath(h.env.root, "./.pony_test/select/ponyup/bin/ponyc"
+      + ifdef windows then ".bat" else "" end)
```

Comment:
i think the string construction should be moved out of the FilePath call.

Reply:
Done.